rc/windowing/sway.kak

```diff
@@ -0,0 +1,67 @@
+provide-module sway %{
+
+# Ensure we're actually in Sway
+evaluate-commands %sh{
+    [ -z "${kak_opt_windowing_modules}" ] || 
+    [ "$DESKTOP_SESSION" = "sway" ] || 
+    [ "$XDG_SESSION_DESKTOP" = "sway" ] || 
```

Comment:
I'm running sway version 1.7 and neither of those `*DESKTOP*` variables is set.
How about using `[ -n "$SWAYSOCK" ]` instead?